src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -41,4 +34,64 @@
       <Link>Shared\Client.cs</Link>
     </Compile>
   </ItemGroup>
+
+  <Target Name="LayoutDependencies" BeforeTargets="Build" Condition="'$(IsInnerBuild)' != 'true'">
```

Comment:
Weird MSBuild way to make things run a target AfterBuild

Reply:
What's wrong with `AfterTargets="AfterBuild"`?

Reply:
Apparently this is the way to run something after build - https://github.com/Microsoft/msbuild/issues/1680#issuecomment-297450755